Sources/SwiftDocC/Indexing/RenderBlockContent+TextIndexing.swift

```diff
@@ -11,53 +11,55 @@
 extension RenderBlockContent: TextIndexing {
     public var headings: [String] {
         switch self {
-        case .heading(_, let text, _):
-            return [text]
+        case .heading(let h):
+            return [h.text]
         default:
             return []
         }
     }
 
     public func rawIndexableTextContent(references: [String : RenderReference]) -> String {
         switch self {
-        case let .aside(_, blocks):
-            return blocks.rawIndexableTextContent(references: references)
-        case let .orderedList(items):
-            return items.map {
+        case let .aside(a):
```

Comment:
Nit: We don't have a lot of single-letter variable names in the codebase thus far. I think it probably fits in a little better to spell out the name of these types instead?

```suggestion
        case let .aside(aside):
```